sdk/devcenter/developer-devcenter-rest/recordings/node/deployment_environments_operations_tests/recording_create_and_then_delete_deployement_environment.json

```diff
@@ -0,0 +1,175 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://8ab2df1c-ed88-4946-a8a9-e1bbb3e4d1fd-sdk-dc-4qb444onnk2tc.centraluseuap.devcenter.azure.com/projects/sdk-project-x2z4o4ohvwcbi/users/me/environments/envname?api-version=2023-04-01",
```

Comment:
I noticed the env vars for playback being updated in the test file.. I think this recording won't work as expected.

Did you run the tests in playback and made sure they are good? 🙂